public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -104,22 +104,28 @@ export class LokiDatasource
   }
 
   getLogsVolumeDataProvider(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> | undefined {
-    const isLogsVolumeAvailable = request.targets.some((target) => target.expr && !isMetricsQuery(target.expr));
+    const isQuerySuitable = (query: LokiQuery) => {
+      const normalized = getNormalizedLokiQuery(query);
+      const { expr } = normalized;
+      // it has to be a logs-producing range-query
+      return expr && !isMetricsQuery(expr) && normalized.queryType === LokiQueryType.Range;
```

Comment:
Hi @gabor , I upgrade grafana from 9.0.2 to 9.1.0, and the loki-like datasoruce query logvolume broken, could you help me look it when you are convenient? thank you.
the expr is '1=1'
![image](https://user-images.githubusercontent.com/7621241/185789869-8393747a-f7cd-4307-b02e-e2d60eb089cf.png)

Reply:
hi @hanjm , please create a new separate github issue for this problem, and make sure to explain:

- what exactly is the problem
- how can this be reproduced

thanks!

Reply:
sorry, i have open a issue for this , thank you.
https://github.com/grafana/grafana/issues/54017